backend-server/src/api/controllers/availabilityController.ts

```diff
@@ -38,3 +71,83 @@ export async function getAvailability(
     startTime
   );
 }
+
+export async function getAllAvailabilities(
+  organization: string,
+  interviewerUID: string
+) {
+  return await dataAccess.getAllAvailabilities(organization, interviewerUID);
+}
+
+export async function getAllCalendarAvailabilities(
+  organization: string,
+  interviewerUID: string
+): Promise<CalendarAvailablity[]> {
+  const availabilities = (await dataAccess.getAllAvailabilities(
+    organization,
+    interviewerUID
+  )) as Availability[];
+  return makeMultipleCalendarAvailabilities(availabilities);
+}
+
+export function makeMultipleAvailabilities(
+  calendarAvailabilities: CalendarAvailablity[]
+): Availability[] {
+  const availabilities: Availability[] = [];
+  for (const calendarAvailability of calendarAvailabilities) {
+    availabilities.push(makeSingleAvailability(calendarAvailability));
+  }
+
+  return availabilities;
+}
+
+export function makeMultipleCalendarAvailabilities(
+  availabilities: Availability[]
+): CalendarAvailablity[] {
+  const calendarAvailabilities: CalendarAvailablity[] = [];
+  for (const availability of availabilities) {
+    calendarAvailabilities.push(makeSingleCalendarAvailability(availability));
+  }
+
+  return calendarAvailabilities;
+}
+
+function makeSingleAvailability(
+  calendarAvailability: CalendarAvailablity
+): Availability {
+  const startDate: Date = new Date(calendarAvailability.start);
```

Comment:
what happens if the `CalendarAvailability` interval is greater than the `Availability` duration? we want the `Availability` duration to be hardcoded based on the organization's parameter right?